benchmarks/benchmark.py

```diff
@@ -62,6 +61,15 @@ def benchmark(golden_rules, tokenize_func):
 
     return percent_score
 
+
+def speed_benchmark(filepath, tokenize_func):
```

Comment:
This benchmark is not right, because you are also measuring the time it takes to read in the file. You need to read in the file _first_ and then segment it separately for each method.

Reply:
Fixed in new script